components/UI/navbar.tsx

```diff
@@ -211,7 +211,7 @@ const Navbar: FunctionComponent = () => {
                 <div className="">
                   <Link href="/">
                     <img
-                      src="/visuals/starknetIdLongLogo.webp"
+                      src="/visuals/starknetIDLongLogo.webp"
```

Comment:
The logo is not showing in the mobile menu bar, I think it should be `starknetIdLongLogo` instead of `starknetIDLongLogo`